### ISSUE TITLE
Can't download Yatopia jar

### ISSUE THREAD

**Issue by cobaltgit** [tr7zw/YAPFA#488]
Hello again. I'm trying to set up a Yatopia MC server on Docker, but I can't seem to download the Yatopia jar, due to 404 errors. Here are my Docker logs
```
[init] Running as uid=1000 gid=1000 with /data as 'drwxrwxr-x    8 1000     1000          4096 May 14 18:42 /data',
[init] Resolved version given LATEST into 1.16.5,
[init] Resolving type given YATOPIA,
[init] Downloading Yatopia from https://api.yatopiamc.org/v2/stableBuild/download?branch=ver/1.16.5 ...,
curl: (22) The requested URL returned error: 404,
[init] ERROR: failed to download from https://api.yatopiamc.org/v2/stableBuild/download?branch=ver/1.16.5 (status=0),
```

**Comment by Aberdeener** [comment]
That's fine. Shouldn't be using a fork as unstable as yatopia anyways.
Check out https://papermc.io :)

**Comment by cobaltgit** [comment]
Oh looks like a duplicate issue

**Comment by cobaltgit** [comment]
Oh I'm using Tuinity atm, should I stick with that or go to PaperMC @Aberdeener

**Comment by tadhgboyle** [comment]
Oh sweet, even better. Tuinity is great!
Have a nice day

**Comment by cobaltgit** [comment]
You too

**Comment by Titaniumtown** [comment]
> That's fine. Shouldn't be using a fork as unstable as yatopia anyways.
> Check out https://papermc.io :)

This is github issues, not a place to recommend other server softwares.

**Comment by Titaniumtown** [comment]
> Hello again. I'm trying to set up a Yatopia MC server on Docker, but I can't seem to download the Yatopia jar, due to 404 errors. Here are my Docker logs
> 
> ```
> [init] Running as uid=1000 gid=1000 with /data as 'drwxrwxr-x    8 1000     1000          4096 May 14 18:42 /data',
> [init] Resolved version given LATEST into 1.16.5,
> [init] Resolving type given YATOPIA,
> [init] Downloading Yatopia from https://api.yatopiamc.org/v2/stableBuild/download?branch=ver/1.16.5 ...,
> curl: (22) The requested URL returned error: 404,
> [init] ERROR: failed to download from https://api.yatopiamc.org/v2/stableBuild/download?branch=ver/1.16.5 (status=0),
> ```

Just download the latest jar, we haven't re-setup the `stableBuild` api yet iirc.

**Comment by basisbit** [comment]
any time frame for when API requests for `stableBuild` might be fixed? If not, should I create a PR to the api repo to either remove stableBuild or make it just HTTP 302 forward to `latestBuild`?
